sci-electronics/drahnr-oregano/drahnr-oregano-0.84.39.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
```

Comment:
does not it really support python 3.7?

Reply:
no, build with 3.7 broken. 3_{4,5,6} work ok

Reply:
This could be fixed by a waf binary update on my end, the version included in the repo is abut 2 years old and at that time iirc there was no python 3.7.

Reply:
waf 2.0.11 work OK with python 3.7